Core/SimpleCurrencyPlugin.lua

```diff
@@ -86,16 +86,16 @@ function L:CreateSimpleCurrencyPlugin(params)
 	-----------------------------------------------
 	local function GetButtonText()
 		local AddSeparator = TitanGetVar(params.titanId, "AddSeparator")
-		local currencyCountText = TitanUtils_GetHighlightText(AddSeparator and BreakUpLargeNumbers(currencyCount) or (currencyCount or "0"))
+		local currencyCountTextNoColor = AddSeparator and BreakUpLargeNumbers(currencyCount) or (currencyCount or "0")
+		local currencyCountText = TitanUtils_GetHighlightText(currencyCountTextNoColor)
 		if (currencyCount or totalSeasonalEarned) and currencyMaximum > 0 then
-			local maxCheckCurrency = (useTotalEarnedForMaxQty and totalSeasonalEarned) or currencyCount
-			currencyCountText = AddSeparator and BreakUpLargeNumbers(currencyCount) or (currencyCount or "0")
+			local maxCheckCurrency = (useTotalEarnedForMaxQty and totalSeasonalEarned and totalSeasonalEarned > 0) or currencyCount
```

Comment:
This is incorrect.

`maxCheckCurrency` was a value, not a boolean; the change makes it a boolean when `totalSeasonalEarned` is > 0

Reply:
yep. that line was only a test and shouldn't have been committed.